Cannot approve call with error "No callData found or supplied"

**Prerequisites**

- Network: Khala Network
- Multisig account (traditional multisig): `3zkooCHZAiukUCPp1dXwTRkRwyfzcyzFFZ1tptXwZShnFoPx`
- Hash: `0x4e3a87dafea28bd17e3b2eb23bf00c5929cc7479bf98471efb2aef27c334f4ee`

**Steps To Reproduce**

Add clear steps on how to reproduce the bug

1. Go to Multix at 'https://multix.chainsafe.io/'
2. Switch to 'Khala' blockchain
3. Find the multisig wallet (traditional, but proxy) and approve the pending transaction
4. Paste the raw call data, see the decoded call, and then click "approve"

**Expected Behaviour**
Send a transaction to approve the multisig call

**Console Error**
```
index-c925ddcd.js:431 No local name to load
(anonymous) @ index-c925ddcd.js:431
index-c925ddcd.js:818 No callData found or supplied
(anonymous) @ index-c925ddcd.js:818
react_devtools_backend_compact.js:2367 Error: createType(Call):: Call: Cannot decode value 'Ea4VwCugA!' of type string
    at createTypeUnsafe (index-c925ddcd.js:430:197059)
    at TypeRegistry.createType (index-c925ddcd.js:430:334294)
    at ApiPromise.createType (index-c925ddcd.js:431:3861)
    at index-c925ddcd.js:818:250
    at Rj (index-c925ddcd.js:40:26649)
    at Ik (index-c925ddcd.js:40:46636)
    at Xk (index-c925ddcd.js:40:45329)
    at Qk (index-c925ddcd.js:40:44293)
    at Fk (index-c925ddcd.js:40:40567)
    at jg (index-c925ddcd.js:38:3538)
```

Inspecting the `rt` object:

```
{
    "hash": "0x4e3a87dafea28bd17e3b2eb23bf00c5929cc7479bf98471efb2aef27c334f4ee",
    "name": "Unknown call",
    "args": false,
    "info": {
        "when": {
            "height": 4197710,
            "index": 19
        },
        "deposit": 1670000000000,
        "depositor": "44RqSGRNvc22mLte4AdrjYkEDurnboQ9N4MFge5F2FHpNnqf",
        "approvals": [
            "44RqSGRNvc22mLte4AdrjYkEDurnboQ9N4MFge5F2FHpNnqf"
        ]
    },
    "from": "3zkooCHZAiukUCPp1dXwTRkRwyfzcyzFFZ1tptXwZShnFoPx",
    "timestamp": "2023-07-03T07:12:30.342Z"
}
```

**Screenshots / Video**
If applicable, please add screenshots or video to help explain or demonstrate the problem you are experiencing.

**Test Host Specifics: **

- OS: macOS 13
- Browser: Chrome
- Version: 114
- Wallet: Polkadot.js

--- a/packages/ui/src/components/modals/ProposalSigning.tsx
+++ b/packages/ui/src/components/modals/ProposalSigning.tsx
@@ -131,7 +131,7 @@ const ProposalSigning = ({
         return
       }
 
-      if (!proposalData.callData || addedCallData) {
+      if (!proposalData.callData && !addedCallData) {
         const error = 'No callData found or supplied'
         console.error(error)
         setErrorMessage(error)

